src/App.tsx

```diff
@@ -1,11 +1,20 @@
-import React from 'react';
-
-import './App.css';
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+//Components
+import MainPage from "./pages/MainPage";
```

Comment:
next step for imports
try to add some configuration for  ur `tsconfig.json`. And then if u use some import  that are inside src folder, it should looks like `pages/MainPage`  without `./`